bochs/bx_debug/parser.y

```diff
@@ -459,6 +463,32 @@ symbol_command:
       }
     ;
 
+set_magic_break_points_command:
+      BX_TOKEN_SET_MAGIC_BREAK_POINTS '\n'
+      {
+        bx_dbg_set_magic_bp_mask(0);
+        free($1);
+      }
+    | BX_TOKEN_SET_MAGIC_BREAK_POINTS BX_TOKEN_STRING '\n'
+      {
+        bx_dbg_set_magic_bp_mask(bx_dbg_get_magic_bp_mask_from_str($2));
+        free($1); free($2);
+      }
+    ;
+
+clr_magic_break_points_command:
+      BX_TOKEN_CLEAR_MAGIC_BREAK_POINTS '\n'
+      {
+        bx_dbg_set_magic_bp_mask(0);
```

Comment:
this clear all magic breakpoints, match your expectations ?

Reply:
yes, without args both commands clear all because who needs enable all? I documented this in doc and help

Reply:
![image](https://github.com/bochs-emu/Bochs/assets/9882181/9dde8913-38d6-4e41-8273-b4d748c9ee02)